valohai_cli/commands/execution/run/dynamic_run_command.py

```diff
@@ -92,6 +93,8 @@ def __init__(
         self.download_directory = download_directory
         self.title = title
         self.environment_variables = dict(environment_variables or {})
+        if environment_variable_groups:
+            self.environment_variable_groups = list(environment_variable_groups)
```

Comment:
AISI, this would leave the instance field possibly unset?

```suggestion
        if environment_variable_groups:
            self.environment_variable_groups = list(environment_variable_groups)
        else:
            self.environment_variable_groups = None
```

It doesn't actually matter in this case since we end up doing `getattr(self, "environment_variable_groups", None)`, but it's good hygiene...